MacrobenchmarkSample/app/build.gradle.kts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("com.android.application")
+    id("kotlin-android")
+}
+
+android {
+    compileSdk = 33
+    namespace = "com.example.macrobenchmark.target"
+
+    defaultConfig {
+        applicationId ="com.example.macrobenchmark.target"
+        versionCode = 1
+        versionName = "1.0"
+        minSdk = 23 // Macrobenchmark doesn"t work with SDK lower than 23
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildFeatures {
+        viewBinding = true
+        compose = true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.3.2"
```

Comment:
It's not. The compiler library is not included in the Compose BOM because its version is independent from the Compose library versions and is coupled to Kotlin versions instead.

Reply:
Thanks @JolandaVerhoef